### PR TITLE
Feature/fix cv es

### PR DESCRIPTION
**Pull request by ag-ramachandran**
#### Pull Request Description

Fix vulns in the pom
Add build and release pipelines

### CODE REVIEW

**Review by tanmaya-panda1 — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -226,6 +263,10 @@
                     <groupId>io.netty</groupId>
                     <artifactId>*</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>io.netty</groupId>
+                    <artifactId>*</artifactId>
```
Comment (asaharn) [inline pom.xml:268]:
```suggestion
                    <groupId>io.netty</groupId>
                    <artifactId>*</artifactId>
                </exclusion>
```

Added twice

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    strategy:
+      matrix:
+        java: [ '8' ]
+    name: Java ${{ matrix.java }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Java 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: 8
```
Comment (asaharn) [inline .github/workflows/release.yml:18]:
```suggestion
          java-version: ${{ matrix.java }}
```

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ '8' ]
+    name: Java ${{ matrix.java }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Java 8
```
Comment (asaharn) [inline .github/workflows/release.yml:15]:
```suggestion
      - name: Setup Java ${{ matrix.java }}
```

File: .github/workflows/release.yml (hunk 2 above)
Comment (asaharn) [inline .github/workflows/release.yml:18]:
```suggestion
          java-version: ${{ matrix.java }}
```

**Pull request merged**